addons/point_of_sale/static/tests/pos/tours/product_screen_tour.js

```diff
@@ -661,3 +661,28 @@ registry.category("web_tour.tours").add("test_product_create_update_from_fronten
             Chrome.endTour(),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("AllowRefreshInOfflineMode", {
+    steps: () =>
+        [
+            Chrome.startPoS(),
+            Dialog.confirm("Open Register"),
+            OfflineUtil.setOfflineMode(),
+            refresh(),
+            ProductScreen.clickDisplayedProduct("Desk Organizer", true, "1", "5.10"),
+            ProductScreen.totalAmountIs("5.10"),
+            ProductScreen.clickPayButton(),
+            PaymentScreen.clickPaymentMethod("Bank", true, { remaining: "0.00" }),
+            PaymentScreen.clickValidate(),
+            ReceiptScreen.isShown(),
+            ReceiptScreen.clickNextOrder(),
+            OfflineUtil.setOnlineMode(),
+            ProductScreen.clickDisplayedProduct("Desk Organizer", true, "1", "5.10"),
```

Comment:
insert a step here that checks that the first order isn't synched to db yet. Something like:

```suggestion
            {
                trigger: ".product-screen",
                run: await () => {
                    const orm = posmodel.env.services.orm;
                    const syncedOrders = orm.read("pos.order", ...);
                    if (syncedOrders.length > 0) {
                        throw new Error("There shouldn't be any order in the db at this point.");
                    }
                }
            },
            ProductScreen.clickDisplayedProduct("Desk Organizer", true, "1", "5.10"),
```